index.html

```diff
@@ -75,5 +75,251 @@ <h4>LETS CONNECT</h4>
+                  <div class="project-details">
+                    <div>
+                      <h1>Tonic</h1>
+                    </div>
+                    <div class="canopy-para">
+                      <p><span>CANOPY</span> <img src="./assets/Counter.png" alt="a picture dot"> Back end dev <img src="./assets/Counter.png" alt="a picture dot"> 2015</p>
+                    </div>
+                    <div class="mini-description">
+                      <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                      </p>
+                    </div>
+                    <div class="language-tags">
+                      <ul>
+                        <div>
+                          <li>html</li>
+                        </div>
+                        <div>
+                          <li>css</li>
+                        </div>
+                        <div>
+                          <li>javaScript</li>
+                        </div>
+                      </ul>
+                    </div>
+                    <div class="project-detail-btn">
+                      <button>See Project</button>
+                    </div>
+                  </div>
+                </div>
+             </div>
+             <div class="project-item">
+                <div class="project-image">
+                  <img src="./assets/fouth-image.png" alt="fourth project image"/>
+                </div>
+                <div>
+                  <div class="project-details">
+                    <div>
+                      <h1>Multi-Post <br/> Stories</h1>
+                    </div>
+                    <div class="canopy-para">
+                      <p><span>CANOPY</span> <img src="./assets/Counter.png" alt="a picture dot"> Back end dev <img src="./assets/Counter.png" alt="a picture dot"> 2015</p>
+                    </div>
+                    <div class="mini-description">
+                      <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                      </p>
+                    </div>
+                    <div class="language-tags">
+                      <ul>
+                        <div>
+                          <li>html</li>
+                        </div>
+                        <div>
+                          <li>css</li>
+                        </div>
+                        <div>
+                          <li>javaScript</li>
+                        </div>
+                      </ul>
+                    </div>
+                    <div class="project-detail-btn">
+                      <button>See Project</button>
+                    </div>
+                  </div>
+                </div>
+             </div>
+       </section>
+      <section class="about-myself-section">
+        <div class="about-myself-description">
+          <div>
+            <h1>About <br/> Myself</h1>
```

Comment:
- [ ] Please use an `h2` tag for this title as the `h1` tag is just reserved for the main title of the webpage. 🤓 
This is for semantics reasons.